packer/ubuntu-14.04_amd64-droplet.json

```diff
@@ -56,11 +53,11 @@
     }
   ],
   "push": {
-    "name": "capgemini/apollo-mesos-ubuntu-14-04-amd64"
+    "name": "Enxebre/apollo-mesos-ubuntu-14-04-amd64"
```

Comment:
We do already version the vagrant box, but the version corresponds to the mesos version at present. And in the vagrantfile we do -

```config.vm.box_version = conf['mesos_version']```